bundle-workflow/src/test_workflow/integ_test/integ_test_suite.py

```diff
@@ -87,7 +92,13 @@ def _execute_integtest_sh(self, endpoint, port, security):
         if os.path.exists(script):
             cmd = f"sh {script} -b {endpoint} -p {port} -s {str(security).lower()}"
             (status, stdout, stderr) = execute(cmd, self.repo.dir, True, False)
+            results_dir = os.path.join(
+                self.repo.dir, "integ-test", "build", "reports", "tests", "integTest"
+            )
+            self.test_recorder.record_test_outcome(
+                self.component.name, self.individual_config, status, stdout, stderr, walk(results_dir)
```

Comment:
`self.individual_config` should come from input, otherwise it's the last item in `self.test_config.integ_test["test-configs"]:`, which is probably not what's intended

Reply:
Done